system_tests/test_vsphere_local_blueprints/local_linux_test.py

```diff
@@ -388,9 +761,37 @@ def cleanup_naming(self):
             task_retry_interval=3,
         )
 
+    def cleanup_naming_underscore(self):
```

Comment:
could all these `cleanup_*` methods be refactored, as they are all the same, except upon which attr they act? e.g.:
```python
def generic_cleanup(component):
    component.execute('uninstall', task_retries=50, task_retry_interval=3)
...
self.addCleanup(functools.partial(generic_cleanup, self.naming_underscore_env))
```